packages/chain-adapters/src/utxo/UTXOBaseAdapter.ts

```diff
@@ -43,7 +59,7 @@ export interface ChainAdapterArgs {
     ws: unchained.ws.Client<unchained.bitcoin.BitcoinTx>
```

Comment:
even though the unchained-clients are essentially the same, i feel like we should still have a union of `unchained.bitcoin` and `unchained.dogecoin` until further utxo abstractions are made to keep pattern consistent across base adapters

Reply:
IMO we shouldn't have separate anything for dogecoin/other utxos. they're all identical aside from configs and the chain they operate on.

Reply:
Agreed long term, just trying to maintain patterns until we can update the abstraction.

Reply:
this ok for now?